[FIX] Fixed URL issue (#209)
tcet-opensource/documentation: docs/projects/tcet-linux-website/about-tcet-linux-website.mdx

description: Official documentation of TCET Linux Website 
---

> >_This is the official documentation of the **[TCET Linux website](https://linux.tcetmumbai.in/)** cell of **[TCET Mumbai](https://tcetmumbai.in)**, maintained by the **[TCET Open Source](https://opensource.tcetmumbai.in)** Documentation team._

Reviewer: I believe it is redundant to copy the description of T&P Website for this project as both projects are created for different purposes. Here, we have used the word `cell` which is irrelevant to TCET Linux Website.
```suggestion
>_This is the official documentation of the **[TCET Linux Website](https://linux.tcetmumbai.in/)** for **TCET Linux**: a GNU/Linux distribution, maintained by the **[TCET Open Source](https://opensource.tcetmumbai.in)** Documentation team._
```

You can even mention the [Teams](https://github.com/orgs/tcet-opensource/teams/devops-team) that have built the project.

P.S: I'm aware this is beyond the scope of the issue, but adding these changes can be fruitful for the overall documentation.
